Index: src/main/java/org/example/User.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/example/User.java b/src/main/java/org/example/User.java
--- a/src/main/java/org/example/User.java	
+++ b/src/main/java/org/example/User.java	
@@ -29,7 +29,6 @@
 
     public static void addToUserAccountBalance(double amount, String currency) {
 
-
         switch (currency) {
 
             case "PLN":
